### ISSUE TITLE
[Bug]: Crash at 8pm Daily

### ISSUE THREAD

**Issue by sparkleys** [homebridge-eufy-security/plugin#113]
### What happened?

Crashing at 8pm daily. I've placed in a child bridge so that it restarts on its own and doesn't affect the rest of my homebridge plugins

HomeBridge Version
v1.5

Operating System
UNRAID 6.9.2

### Device Type

Doorbell (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

other (UnSupported)

### NodeJS Version

v16 (Supported)

### Operating System

Docker (UnSupported)

### Relevant log output

```shell
[7/16/2022, 8:00:08 PM] [EufySecurity-2.0.0] INFO: [32mCountry set: US[39m
[7/16/2022, 8:00:08 PM] [EufySecurity-2.0.0] INFO: [32mFinished initializing![39m
[7/16/2022, 8:00:08 PM] Homebridge v1.5.0 (HAP v0.10.2) (homebridge-eufy-security) is running on port 56545.
Error: ENOENT: no such file or directory, rename '/root/.homebridge/eufysecurity/log-lib.log.1' -> '/root/.homebridge/eufysecurity/log-lib.log.2'

[7/16/2022, 8:00:08 PM] [homebridge-eufy-security] Child bridge process ended
[7/16/2022, 8:00:08 PM] [homebridge-eufy-security] Process Ended. Code: 1, Signal: null
[7/16/2022, 8:00:15 PM] [homebridge-eufy-security] Restarting Process...
[7/16/2022, 8:00:15 PM] [homebridge-eufy-security] Launched child bridge with PID 16692
[7/16/2022, 8:00:15 PM] Registering platform 'homebridge-eufy-security.EufySecurity'
[7/16/2022, 8:00:15 PM] [homebridge-eufy-security] Loaded homebridge-eufy-security v2.0.0 child bridge successfully
[7/16/2022, 8:00:16 PM] Loaded 1 cached accessories from cachedAccessories.0E3976DD86C9.
[7/16/2022, 8:00:16 PM] [EufySecurity-2.0.0] WARN: [33mwarning: planned changes, see https://github.com/homebridge-eufy-security/plugin/issues/1[39m
[7/16/2022, 8:00:16 PM] [EufySecurity-2.0.0] INFO: [32mCountry set: US[39m
[7/16/2022, 8:00:16 PM] [EufySecurity-2.0.0] INFO: [32mFinished initializing![39m
[7/16/2022, 8:00:16 PM] Homebridge v1.5.0 (HAP v0.10.2) (homebridge-eufy-security) is running on port 56545.
Error: ENOENT: no such file or directory, rename '/root/.homebridge/eufysecurity/log-lib.log' -> '/root/.homebridge/eufysecurity/log-lib.log.0'

[7/16/2022, 8:00:16 PM] [homebridge-eufy-security] Child bridge process ended
[7/16/2022, 8:00:16 PM] [homebridge-eufy-security] Process Ended. Code: 1, Signal: null
[7/16/2022, 8:00:23 PM] [homebridge-eufy-security] Restarting Process...
[7/16/2022, 8:00:23 PM] [homebridge-eufy-security] Launched child bridge with PID 16705
[7/16/2022, 8:00:23 PM] Registering platform 'homebridge-eufy-security.EufySecurity'
```

**Comment by thieren** [comment]
Hi @sparkleys 

this is a duplicate of #56 and #84 

It's already fixed in the latest beta. Please try.

**Comment by thieren** [comment]
Is fixed with release of 2.1.0
